### DISCUSSION TITLE
Pre-recorded audio files not always fully transcribed

### DISCUSSION THREAD

**Discussion by bbauman1** [deepgram/community#775] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

Since moving from OpenAI whisper to nova-2 we are getting a noticeable increase in support tickets from our users telling us that their audio files are not being fully transcribed. They are uploading pre-recorded files, usually 30-60 minutes in length, and some are seeing large portions of the end of their transcripts missing. 

I'm using the javascript sdk version 3.3.1

### If possible, please attach your code or paste it into the text box.

```
let deepgramData: PrerecordedSchema = {
  model: "nova-2",
  smart_format: true,
  detect_language: true
}

const { result, error } = await deepgram.listen.prerecorded.transcribeFile(
  req.file.buffer,
  deepgramData
);
```

### If possible, please attach an example audio file to reproduce the issue.

Don't have an audio file to attach but they are usually .m4a files of 30-60 minutes in length

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
Hi @bbauman1,

Do you happen to notice if this happens when you upload the same type of file to the [Deepgram playground](https://playground.deepgram.com/?endpoint=listen&smart_format=true&detect_language=true&model=nova-2) to be transcribed? I would like to determine if the issue is the API or if it's with the Javascript SDK.

Sandra

**Reply by bbauman1** [reply]
Hey @SandraRodgers unfortunately I can't test with a user's audio file. I found this issue which sounds like it might be similar https://github.com/orgs/deepgram/discussions/474

Would making direct requests to the deepgram api without using the sdk remove any chance of issues there?

**Reply by SandraRodgers** [reply, marked as answer]
I asked the team and it is something that Deepgram engineers are aware of and currently working on. Unfortunately that's all I can tell you since I don't have a solution for you. There was a recent to reply (from few hours ago) to the thread you linked [here](https://github.com/orgs/deepgram/discussions/474#discussioncomment-9528426) with some suggestions for dealing with the issue. You could try his suggestions and if they work, share that with us (and the community). We hope to have this fixed soon; thank you for providing your report about how it's affecting you.

**Reply by bbauman1** [reply]
ok @SandraRodgers thanks. will updates be posted here or should I be checking for them elsewhere?
